hadoop-tools/hadoop-distcp/src/test/java/org/apache/hadoop/tools/contract/AbstractContractDistCpTest.java

```diff
@@ -532,13 +549,15 @@ private Path distCpDeepDirectoryStructure(FileSystem srcFS,
    */
   private void largeFiles(FileSystem srcFS, Path srcDir, FileSystem dstFS,
       Path dstDir) throws Exception {
+    int fileSizeKb = conf.getInt(SCALE_TEST_DISTCP_FILE_SIZE_KB,
+        DEFAULT_DISTCP_SIZE_KB);
+    if (fileSizeKb < 1) {
+      skip("File size in " + SCALE_TEST_DISTCP_FILE_SIZE_KB + " too small");
```

Comment:
Nit: maybe, we should say to "make fileSize in `SCALE_TEST_DISTCP_FILE_SIZE_KB` be greater than or equal to 1" or "File Size in `SCALE_TEST_DISTCP_FILE_SIZE_KB` smaller than 1"

Reply:
now

"File size in " + SCALE_TEST_DISTCP_FILE_SIZE_KB + " is zero

It's not bug, just a fact...for HDFS suite it'll be zero by default now